tests/core/gas-strategies/test_time_based_gas_price_strategy.py

```diff
@@ -108,21 +117,21 @@ def _get_block_by_something(method, params):
     'strategy_params,expected',
     (
         # 120 second wait times
-        (dict(max_wait_seconds=80, sample_size=5, probability=98), 61),
-        (dict(max_wait_seconds=80, sample_size=5, probability=90), 46),
-        (dict(max_wait_seconds=80, sample_size=5, probability=50), 40),
+        (dict(max_wait_seconds=80, sample_size=5, probability=98), 70),
+        (dict(max_wait_seconds=80, sample_size=5, probability=90), 23),
+        (dict(max_wait_seconds=80, sample_size=5, probability=50), 7),
         # 60 second wait times
-        (dict(max_wait_seconds=60, sample_size=5, probability=98), 62),
-        (dict(max_wait_seconds=60, sample_size=5, probability=90), 55),
-        (dict(max_wait_seconds=60, sample_size=5, probability=50), 40),
+        (dict(max_wait_seconds=60, sample_size=5, probability=98), 92),
+        (dict(max_wait_seconds=60, sample_size=5, probability=90), 48),
+        (dict(max_wait_seconds=60, sample_size=5, probability=50), 7),
         # 40 second wait times
-        (dict(max_wait_seconds=40, sample_size=5, probability=98), 63),
-        (dict(max_wait_seconds=40, sample_size=5, probability=90), 61),
-        (dict(max_wait_seconds=40, sample_size=5, probability=50), 40),
+        (dict(max_wait_seconds=40, sample_size=5, probability=98), 100),
+        (dict(max_wait_seconds=40, sample_size=5, probability=90), 81),
+        (dict(max_wait_seconds=40, sample_size=5, probability=50), 7),
         # 20 second wait times
-        (dict(max_wait_seconds=20, sample_size=5, probability=98), 63),
-        (dict(max_wait_seconds=20, sample_size=5, probability=90), 62),
-        (dict(max_wait_seconds=20, sample_size=5, probability=50), 50),
+        (dict(max_wait_seconds=20, sample_size=5, probability=98), 100),
+        (dict(max_wait_seconds=20, sample_size=5, probability=90), 100),
+        (dict(max_wait_seconds=20, sample_size=5, probability=50), 34),
```

Comment:
I'm not reviewing these numbers, just trusting that you double-checked them. :)

Reply:
It might be a good idea to at least once check how accurate the target times are when compared to actual times on ropsten or even mainnet.

Reply:
:+1: 

Also, as a quicker test, we could compare against https://ethgasstation.info/.

Reply:
Ooh yeah.  Then I could compare to main net without spending any gas.

Reply:
The estimates are comparable to ethgasstation.info, but I noticed that with longer acceptable wait times we were a bit lower, so I ran some tests by sending a transaction back and forth on on mainnet using our estimate.  I tried raising the percentile.  Given the results, I am going to bump up the percentile to 20%, since it looks the best.

15% percentile
| Expected        | Actual           |
| ------------- |:-------------:|
| <2m      | 2m |
| <5m      | 10m, 4m      |
| <30m | 13m      |


20% percentile
| Expected        | Actual           |
| ------------- |:-------------:|
| <2m      |1.5m  |
| <5m      | 2m, 4m, 3m      |
| <30m | 15m      |

25%
| Expected        | Actual           |
| ------------- |:-------------:|
| <2m      | 2m |
| <5m      | 2m      |
| <30m | 3m      |

40%
| Expected        | Actual           |
| ------------- |:-------------:|
| <30m | 7m      |

Reply:
Solid :+1: